Python/Stream/pyStream.cpp

```diff
@@ -388,7 +382,6 @@ PY_METHOD_VA(Stream, writeLine, nullptr)
 }
 
 static PyMethodDef pyStream_Methods[] = {
-    pyStream_close_method,
```

Comment:
This probably requires a corresponding update to the type hints for `hsFileStream` and `hsStream` in PyHSPlasma.pyi.

Reply:
Ah, good point